packages/elements/src/clock/__test__/clock.interactive.test.js

```diff
@@ -39,75 +47,79 @@ describe('clock/Interactive', () => {
       elem.dispatchEvent(keyboardEvent('keyup', keyOption));
     };
 
+    const updateBtnSelector = () => {
+      incrementBtnInHours = hoursSegment.querySelector('[part=increment-button]');
+      decrementBtnInHours = hoursSegment.querySelector('[part=decrement-button]');
+      incrementBtnInMinutes = minutesSegment.querySelector('[part=increment-button]');
+      decrementBtnInMinutes = minutesSegment.querySelector('[part=decrement-button]');
```

Comment:
A function referencing variables `hoursSegment` outside is bad. Please pass `el` into this function.